src/libs/actions/PaymentMethods.ts

```diff
@@ -214,7 +214,7 @@ function clearDebitCardFormErrorAndSubmit() {
     Onyx.set(ONYXKEYS.FORMS.ADD_DEBIT_CARD_FORM, {
         isLoading: false,
         errors: undefined,
-        setupComplete: false,
```

Comment:
@teneeto Why are we changing this to true?

Reply:
apologies @abdulrahuman5196 just seeing this, let me have a look.

Reply:
@abdulrahuman5196, I've reverted that change, that was a wrong code update. Thanks for Spotting it.